lib/chat/chat.dart

```diff
@@ -1,18 +1,72 @@
+import 'package:google_fonts/google_fonts.dart';
 import 'package:flutter/material.dart';
+import './new_matches.dart';
+import './messages.dart';
 
-// TODO: Update this placeholder widget
 class Chat extends StatelessWidget {
   const Chat({Key key}) : super(key: key);
 
   @override
   Widget build(BuildContext context) {
-    return Scaffold(
-      backgroundColor: Colors.black,
-      body: Container(
-        child: Center(
-          child: Text("Matches", style: TextStyle(color: Colors.white, fontSize: 30.0))
-        )
-      )
+    return Stack(
```

Comment:
In the `Stack`'s children, you're returning one widget -- `Scaffold`. You could get rid of the `Stack` and move up `Scaffold`.

Reply:
I just wanted to leave this here for in the future when we have an appropriate background image

Reply:
Discussed offline and decided it's fine. 👍